tests/queue/test_file.py

```diff
@@ -0,0 +1,83 @@
+'''
+Unit tests for `tilequeue.queue.file`.
+'''
+
+from tilequeue.queue import OutputFileQueue
+from tilequeue import tile
+from ModestMaps.Core import Coordinate
+import unittest
+import StringIO
+
+
+class TestQueue(unittest.TestCase):
+
+    def setUp(self):
+        self.test_tile_coords = [
+            (0, 0, 0),
+            (1, 2, 3),
+            (4, 5, 6),
+            (9, 3, 1),
+            (4, 7, 1)
+        ]
+        self.test_tile_objs = [
+            Coordinate(*coords) for coords in self.test_tile_coords]
```

Comment:
I think it's better to specify the column, row, zoom explicitly via the keyword properties, and this goes for the other instantiations too, if only to make it clear to the reader. The default order that we get from modest maps is imho unexpected, namely row, column, zoom. It doesn't look like it's what you expected either, since there's some invalid coordinates specified. In this case it doesn't matter though, because there aren't checks for invalid coordinates, but it can still lead to confusion.